app/components/ui/Banner/Banner.scss

```diff
@@ -2,16 +2,25 @@
 
 .bannerContainer {
   width: 100%;
-  padding: calc-em(20px) calc-em(30px);
+  padding: calc-em(16px) calc-em(28px);
   color: black;
   background-color: $color-yellow;
   font-size: calc-em(24px);
   text-align: center;
+  position: sticky;
```

Comment:
Are you trying to get the banner to always be present at the top of the page & have the rest of the page scroll under it?